src/CMakeLists.txt

```diff
@@ -8,16 +8,14 @@ if(NOT APPLE AND NOT WIN32)
   # parallelism to bleed into code that is executed within the runtime, where
   # USM is a big no-no.
   set(HIPSYCL_STDPAR_RT_LINKER_FLAGS "-Wl,-Bsymbolic-functions")
+  set(base $ORIGIN)
 else()
   set(HIPSYCL_STDPAR_RT_LINKER_FLAGS "")
+  set(base @loader_path)
```

Comment:
What does this do?

Reply:
This is needed for apple. I think it wont have an effect on windows

Reply:
I understand that it's needed for Apple, I was just curious what `@loader_path` means :D